### ISSUE TITLE
This is no completion when filetype=cuda

### ISSUE THREAD

**Issue by wrvsrx** [clangd/coc-clangd#51]
## CocInfo

vim version: VIM - Vi IMproved 8.2 8020717
node version: v14.2.0
coc.nvim version: 0.0.78-42353685a1
term: dumb
platform: linux

## coc-clangd version

0.4.3

## Describe the bug

Completion doesn't show in .cu file.

## Reproduce

1. create compile_flags.txt
```
-xcuda
-std=c++11
--cuda-path=/opt/cuda
--cuda-gpu-arch=sm_50
-I./include
-I/opt/cuda/include
```
2. open test.cu and there's no completion
3. :set filetype=cpp
4. completion appears and it can complete cuda syntax normally

**Comment by fannheyward** [comment]
What's the filetype when you open a `cuda` file? You can get it by `:verbose set filetype`

**Comment by wrvsrx** [comment]
cuda. I get following image by exec "echo &filetype"
![image](https://user-images.githubusercontent.com/42770726/82511915-17a39f00-9b41-11ea-9270-d6050085a013.png)

**Comment by fannheyward** [comment]
Reproduced, coc-clangd is activated on cuda files, but no completions. Add follow settings to your vimrc as a workround:

```viml
let g:coc_filetype_map = {
      \ 'cuda': 'cpp',
      \ }
```

@sam-mccall does this need any other settings for CUDA files?

**Comment by fannheyward** [comment]
VSCode + clangd work fine for CUDA files, will look into this issue.

**Comment by fannheyward** [comment]
Fixed in v0.4.4
